README.md

```diff
@@ -20,5 +20,8 @@ List of the [GitHub](https://github.com) usernames of all the people interested
 - [Aya Salama] (https://github.com/Aya-S)
 - [Bhagya C](https://github.com/BhagyaC)
 
-
 This list is not yet complete—we'll keep adding more people.
+
+## Lessons Learned:
+Please use the link below to review or capture key lessons learned as new contributors: <br>
```

Comment:
my preference would be to leave a blank line between the paragraphs instead of the `<br>`. But do we want a line break here?